src/Orchestration/Adapter/DockerAPI.php

```diff
@@ -427,6 +429,8 @@ public function run(
 
         $vars = $parsedVariables;
 
+        // TODO: Connect to $network if not empty
```

Comment:
Should we address this comment?

Reply:
I spent quite some time but couldn't figure it out. I think we can lose focus on DocketAPI adapter until we utilize it. There are already some places it doesn't act 100% as it should.